crates/engine/tree/src/tree/payload_processor/multiproof.rs

```diff
@@ -918,15 +928,21 @@ fn get_proof_targets(
     // then process storage slots for all accounts in the state update
     for (hashed_address, storage) in &state_update.storages {
```

Comment:
can we solve this issue by just checking the `storage.wiped == true`, and adding the address to the targets if that's the case?